backend/models/contractorModel.js

```diff
@@ -30,14 +29,14 @@ const contractorSchema = new Schema({
     required: false
   },
   // The services that this contractor provides
-  // TODO: figure out how services will be checked on backend
   services: {
```

Comment:
Do you want to keep this format or `service: {
    type: [String],
    enum: getServices(),
    required: false
  },` I will update my code accordingly then.

Reply:
I updated the code, so I'm not sure what line you are referring to. However, if you're referring to the definition of services, it should be the following:


services: {
    type: [
      {
        type: String,
        enum: getServices()
      }
    ],
    required: false,
  }

The previous definition would not have used the enum validator the way we intended.

Reply:
Ok I will update my CO service def to this.